tests/test_modeling_utils.py

```diff
@@ -262,6 +262,11 @@ def test_find_tied_parameters(self):
         model.block1.linear1.weight = model.block2.linear1.weight
         assert find_tied_parameters(model) == [["block1.linear1.weight", "block2.linear1.weight"]]
 
+        layer = nn.Linear(10, 10)
+        model = nn.Sequential(layer, layer)
+        tied_params = find_tied_parameters(model)
+        assert sorted(tied_params) == [["0.bias", "1.bias"], ["0.weight", "1.weight"]]
+
```

Comment:
Nice test that shows exactly the issue that you are experiencing !

Reply:
Thanks!